src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs

```diff
@@ -484,6 +484,10 @@ private string QuoteName(object name)
         {
```

Comment:
I looked through the calls to QuoteName() and I believe you are right that it's only a string or Uri.  I'll separate it.